lib/cisco_node_utils/cmd_ref/vlan.yaml

```diff
@@ -11,6 +11,16 @@ create:
 destroy:
   config_set: "no vlan %s"
 
+mode:
+  _exclude:
+    - /N9K/
+    - /N3/
```

Comment:
Since it is a regex, hopefully it will work for all N3 variants. I see that the product id is something like N3064.

From: Mike Wiebe <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cisco/cisco-network-node-utils <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Friday, November 13, 2015 at 8:01 AM
To: cisco/cisco-network-node-utils <cisco-network-node-utils@noreply.github.com<mailto:cisco-network-node-utils@noreply.github.com>>
Cc: Cisco Employee <dcherian@cisco.com<mailto:dcherian@cisco.com>>
Subject: Re: [cisco-network-node-utils] adding fabricpath topo and fixed feature_set as a class method (#75)


In lib/cisco_node_utils/cmd_ref/vlan.yaml<https://github.com/cisco/cisco-network-node-utils/pull/75#discussion_r44798265>:

> @@ -11,6 +11,16 @@ create:
>  destroy:
>    config_set: "no vlan %s"
>
> +mode:
> +  _exclude:
> +    - /N9K/
> +    - /N3/


Missing 'K' here? Also, you should be able to exclude these using a single line.

_exclude: [/N3K/, /N9K/]

-
Reply to this email directly or view it on GitHub<https://github.com/cisco/cisco-network-node-utils/pull/75/files#r44798265>.